rust/ql/test/library-tests/dataflow/local/DataFlowStep.ql

```diff
@@ -6,8 +6,6 @@ private predicate provenance(string model) { RustDataFlow::simpleLocalFlowStep(_
 
 private module Tm = TranslateModels<provenance/1>;
```

Comment:
May as well remove this and the `provenance` predicate.

Reply:
Ah, sorry I missed this.  Since the PR has been merged I've made a follow-up for this: https://github.com/github/codeql/pull/19353